cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Insert/Tx.hs

```diff
@@ -0,0 +1,433 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+
+module Cardano.DbSync.Era.Shelley.Insert.Tx (
+  insertTx,
+  insertTxOut,
+) where
+
+import Cardano.BM.Trace (Trace)
+import Cardano.Db (DbLovelace (..), DbWord64 (..))
+import qualified Cardano.Db as DB
+import Cardano.DbSync.Api
+import Cardano.DbSync.Api.Types (InsertOptions (..), SyncEnv (..))
+import Cardano.DbSync.Cache.Types (Cache (..))
+
```

Comment:
Looks like a superfluous blank line here

Reply:
I was splitting between cardano related libraries and standard libraries. Can remove not to worry 👍